Tasks/NuGetInstaller/nugetinstaller.ts

```diff
@@ -33,6 +33,11 @@ async function main(): Promise<void> {
     try {
         tl.setResourcePath(path.join(__dirname, "task.json"));
 
+        // set the console code page to "Unicode"
+        if (process.platform === "win32") {
```

Comment:
This looks good.  I think we should consider doing the same change inside toolrunner.exec.  Eric and I have been discussing.  If you want to do a PR there if you agree, we can take that.